docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -7,6 +7,14 @@
 
 The ServiceNow connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow incidents.
 
+[float]
+[[servicenow-connector-prerequisites]]
+==== Prerequisites
+Create an integration user in ServiceNow and assign it the following roles.
+
+* `incident_manager`: Allows the user read, create, and update access to the Incident table.
```

Comment:
Are Incident, Choice, and Severity labels in the UI?  If so, they should be in bold.